video/out/vo_dmabuf_wayland.c

```diff
@@ -41,6 +41,9 @@
 #include "present_sync.h"
 #include "wayland_common.h"
 #include "generated/wayland/linux-dmabuf-unstable-v1.h"
+#if HAVE_WAYLAND_PROTOCOLS_1_27
+#include "generated/wayland/single-pixel-buffer-v1.h"
+#endif
 #include "generated/wayland/viewporter.h"
```

Comment:
This probably could be organized a bit better. Mind moving all the generated headers down in a separate section with the `#if` in its own section below. Like how `wayland_common.c` does it basically.